### ISSUE TITLE
Component resizing

### ISSUE THREAD

**Issue by szeplaszloit** [alump/GridStack#21]
Hi,

if I have only one component on `GridStackLayout`, I can't vertical resize this component. You can check this issue on the demo site: [http://app.siika.fi/GridStackDemo/](http://app.siika.fi/GridStackDemo/)
Remove all components, except the Form, then try to resize it...

Have you got any idea?

Thanks.

**Comment by alump** [comment]
Yes, I can only change size horizontally. Looks like single item gets limited to height of 1 slot. I will check as soon as I find some time.

**Comment by alump** [comment]
min-height: 100% applied to .v-panel-content > .grid-stack breaks resizing vertically
